### PR TITLE
Update shared fx in preview2.1 to 1.1.0-preview1-001100-00

### PR DESCRIPTION
**Pull request by livarcocc**
@eerhardt @brthor @jgoshi @piotrpMSFT

### CODE REVIEW

File: src/dotnet/project.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     },
     "Microsoft.NETCore.App": {
       "type": "platform",
-      "version": "1.1.0-preview1-001086-00"
+      "version": "1.1.0-preview1-001100-00"
     },
     "System.Diagnostics.TraceSource": "4.0.0",
```
Comment (eerhardt) [inline src/dotnet/project.json]: What about the CoreFX dependencies?  Shouldn't these be updated?  Same goes for any CoreClr dependencies we have.  These should all be updated to the version core-setup is using of the two repos.

**Comment by eerhardt** [comment]
This looks better - thanks!

**Comment by livarcocc** [comment]
@dotnet-bot Test Fedora23 x64 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX x64 Release Build
@dotnet-bot Test OpenSUSE13.2 x64 Debug Build
@dotnet-bot Test Ubuntu16.10 x64 Debug Build
@dotnet-bot Test Windows_NT x64 Release Build
@dotnet-bot Test Windows_NT x86 Debug Build

**Comment by livarcocc** [comment]
@dotnet-bot Test  CentOS7.1 x64 Debug Build
@dotnet-bot Test Ubuntu x64 Release Build

**Pull request merged**